frontend/src/components/SelectedPlaylistInfo.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+interface Props {
+  playlistLink: string;
+}
+
+export const SelectedPlaylistInfo: React.FC<Props> = (props) => {
```

Comment:
```suggestion
export const SelectedPlaylistInfo: React.FC<Props> = ({ playlistLink }) => {
```

prefer destructuring inline if you know what every prop should be. this helps make things more explicit and more readable